docs/cookbook/making_cross_origin_http_requests.md

```diff
@@ -1,38 +1,42 @@
 # Making cross-origin HTTP requests
 
-## Description
+eZ Platform ships with [NelmioCorsBundle](https://github.com/nelmio/NelmioCorsBundle),
+an open-source Symfony bundle that provides support for [CORS (Cross-Origin Resource Sharing)](http://www.w3.org/TR/cors/).
+The REST API is pre-configured to respond to such requests, as long as you customize the allowed origins as explained below.
 
-eZ Publish ships with [NelmioCorsBundle](https://github.com/nelmio/NelmioCorsBundle), a Symfony open-source bundle that provides support for [CORS (Cross Origin Resource Sharing)](http://www.w3.org/TR/cors/). The REST API is pre-configured to respond to such requests, as long as you customize the allowed origins as explained below.
+## What is CORS
 
-### What is CORS
+Supported by most modern browsers, this W3C specification defines a set of custom headers
+that, under specific circumstances, allow HTTP requests between different hosts.
```

Comment:
'allows' as we have 'set of custom headers'?

Reply:
But it's "a set of headers" and each single header can "allow", so singular IMHO